### PR TITLE
Update ultraplex to 1.2.9

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`ultraplex`](https://bioconda.github.io/recipes/ultraplex/README.html): **1.2.5** &rarr; **1.2.9**

[install with bioconda](http://bioconda.github.io/recipes/ultraplex/README.html) [Conda](https://anaconda.org/bioconda/ultraplex/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/ultraplex`](https://github.com//bioconda/bioconda-recipes/tree/bump/ultraplex/recipes/ultraplex) (click to view/edit other files)
Summary | fastq demultiplexer
Home | [https://github.com/ulelab/ultraplex.git](https://github.com/ulelab/ultraplex.git)
Releases |[https://pypi.org/pypi/ultraplex/json](https://pypi.org/pypi/ultraplex/json)
Recipe Maintainer(s) | @Delayed-Gitification, @CharlotteAnne

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by CharlotteAnne** [comment]
@mencian - thanks so much for your help so far, as you can see in my patch I've resolved the initial issue (python <3.10 is required https://github.com/bioconda/bioconda-recipes/pull/44335) but am stumped that setuptools-scm can't find the version in the pypi tarball - I've confirmed that PKG-INFO is pressent in the pypi tarball and contains the version number - do you have any other ideas for how to resolve this?

**Comment by CharlotteAnne** [comment]
@mencian seems all good to go now! thank you!

**Comment by mencian** [comment]
@BiocondaBot please fetch artifacts

**Comment by CharlotteAnne** [comment]
@mencian would you be able to merge? I have some other software to update relying on this update, thank you

**Comment by mencian** [comment]
I apologize for the delay, @CharlotteAnne. There seems to be 3 build artifacts for python `3.9`, which is a little odd, since there should be builds for python `3.8` as well. I'll make a few edits to see if I can get the CI to build for `3.8` and check back.

File: recipes/ultraplex/meta.yaml [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
-  sha256: 13fcb6bcbb20c1765a544869171dff4f31a930b77ebb44bed6a6000c4ea9a3b4
-
+  sha256: 7b1efa09a421590907c7e5177eb9e4837deade89c237b961bcf2f7edfa9f2e90
 
 build:
-  number: 2
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-  skip: True  # [py < 37 or py > 39]
+  number: 0
+  skip: True  # [py < 37]
```
Comment (mencian) [inline recipes/ultraplex/meta.yaml:14]:
```suggestion
  skip: True  # [py < 37 or py > 39]
```

File: recipes/ultraplex/meta.yaml [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
+  run_exports:
+    - {{ pin_subpackage('ultraplex', max_pin="x") }}
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
     - pip
-    - python
+    - python <3.10
```
Comment (mencian) [inline recipes/ultraplex/meta.yaml:27]:
```suggestion
    - python
```

**Comment by mencian** [comment]
@BiocondaBot please fetch artifacts

**Comment by mencian** [comment]
I now see the python `3.8` builds for both Linux and OSX. Thanks for doing the heavy lifting, @CharlotteAnne!

**Review by mencian — APPROVED** [review]

**Comment by CharlotteAnne** [comment]
thanks so much @mencian !!! i have learnt quite a bit now doing this haha - appreciate it!

**Pull request merged**